### ISSUE TITLE
4.3.5 release does not include remote traceback

### ISSUE THREAD

**Issue by stevenheidel** [noxdafox/pebble#19]
Hey @noxdafox  - I noticed that the remote traceback change I made wasn't included in 4.3.5 but is on master, would it be possible to create a release with that change included?

**Comment by noxdafox** [comment]
Hello, I am planning to add some work on top of your PR to make it more generic but I am quite swamped right now as just temporarily relocated to another country.

In the meantime, you can find the remote traceback appended to the exception. I'll make sure changes will be backward compatible so you can consider it as long term solution anyway.

```
try:
    result = future.result()
except Exception as error:
    print(error.traceback)  # traceback as formatted string here
```

**Comment by stevenheidel** [comment]
That sounds good, we've just pinned to the git version for now.

**Comment by noxdafox** [comment]
I am currently looking at the `concurrent.futures` implementation as pebble API are inspired by the ones in that module.

**Comment by noxdafox** [comment]
Fixed in 4.3.6.

Added remote traceback in Python 3 for the following:
* @concurrent.process
* ProcessPool.submit
* ProcessPool.map

Thanks for your contribution!
